arch/arm64/boot/dts/freescale/imx8mp-maivin.dts

```diff
@@ -68,11 +68,23 @@
 	clock-frequency = <400000>;
 	scl-gpios = <&gpio5 16 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
 	sda-gpios = <&gpio5 17 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
+
+	dac@0d {
```

Comment:
Addresses shouldn't have a 0 prefix in node names, this should be dac@d. Same below. The reg property is fine.

Other than that, the change looks good, but could you please expand the commit message, especially to use the correct prefix on the subject line, as explained in the review of the eeprom enablement branch ?